config/locales/en.yml

```diff
@@ -367,9 +367,11 @@ en:
     wait_for_an_email: Wait for an email from GOV.UK Verify confirming your new signing certificate is in use
     how_long_it_takes: This usually takes a few minutes from the time you uploaded the new signing certificate.
     delete_old_signing_key_and_cert: Then you can delete the old signing key and certificate from your %{component} configuration.
-    maintain_connection: You need to manage your certificates to make sure your service stays connected to GOV.UK Verify.
-    connection_broken_or_compromised_guidance: If your connection has been broken or compromised, read
-    connection_broken_or_compromised_link: this guidance
+    maintain_connection_p1: Manage your certificates to make sure your service stays connected to GOV.UK Verify. 
+    maintain_connection_p2: When your certificates are expiring, you can rotate them in any order. You can also rotate more than one certificate at the same time.
```

Comment:
I'd have preferred these combined as one html block, but not gonna fuss about that now